### PR TITLE
check if config exists before writing to it

### LINKED ISSUE #186302: Accessibility View: accessibility.verbosity.editor is not a registered configuration
Testing #186214

1. Open the accessibility view from Copilot Chat
2. Press "D" to disable accessibility verbosity
3. Notice error notification `Unable to write to User Settings because accessibility.verbosity.editor is not a registered configuration.`

I noticed that the `accessibility.verbosity.panelChat` still gets nicely written.

Screenshot 2023-06-27 at 11 55 46

### PR DESCRIPTION
**Pull request by meganrogge**
fixes https://github.com/microsoft/vscode/issues/186302

### CODE REVIEW

File: src/vs/workbench/contrib/accessibility/browser/accessibleView.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ class AccessibleView extends Disposable {
 			this._register(this._editorWidget.onKeyUp((e) => {
 				if (e.keyCode === KeyCode.Escape) {
 					this._contextViewService.hideContextView();
-				} else if (e.keyCode === KeyCode.KeyD) {
+				} else if (e.keyCode === KeyCode.KeyD && this._configurationService.getValue(settingKey)) {
```
Comment (connor4312) [inline src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:129]: Does reading a configuration key on every keyup even have a performance overhead?

Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:129]: only when the key that's up is `D`

Comment (connor4312) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:129]: oh, nevermind, this is only the accessible view

**Review by connor4312 — APPROVED** [review]

File: src/vs/workbench/contrib/accessibility/browser/accessibleView.ts (hunk 1 above)
Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:129]: which will be rarely in a readonly editor 😄

**Pull request merged**